zubhub_frontend/zubhub/package.json

```diff
@@ -23,10 +23,10 @@
     "react": "^17.0.1",
     "react-dom": "^17.0.1",
     "react-i18next": "^11.8.5",
-    "react-quill": "^1.3.5",
+    "react-quill": "^0.0.2",
```

Comment:
Any reason, why we downgraded this package?

Reply:
in order to access auto update on save i used npm to start the server on my terminal it was downgraded when i ran npm install command because it wasn't compatible with my version of npm. but anyother person who uses npm start on the frontend it's going to install the packages compatible with their version.

Reply:
@Morai-C , perhaps you should upgrade your npm version?